src/components/SiteHeader/index.jsx

```diff
@@ -18,6 +20,15 @@ const getCurrentPage = (pageId, aboutPageId, indexPageId) => {
   return matchingRoutes.length > 0 ? matchingRoutes[0].id : null;
 };
 
+const renderLogo = (className, siteLogo, defaultImage, altText) => (
```

Comment:
Can you make this a functional component instead? I think `renderBlah` methods are usually only used as a private function of a class component. Seems a bit off to be passing it around as is. 

Could also move it into it's own file since `Header` and `Footer` both use it (that was `Footer` isn't dependent on `Header`), but I'm less concerned about that.

Reply:
@tuchfarber 
made it a function component and moved to it's own file